client/src/components/CommentApp/components/Comment/index.tsx

```diff
@@ -629,8 +629,22 @@ export default class CommentComponent extends React.Component<CommentProps> {
                 }),
               );
             },
+            // Allow delay for side panel to open and comment card to fade in with animations
+            checkCanFocusTrap: (containers: (HTMLElement | SVGElement)[]) => {
+              const hasFocusTarget = containers.some(
+                (container) =>
+                  container.getAttribute('data-focus-target') === 'true',
```

Comment:
This gets us pretty close, but it still does not focus when there is an existing comment AND the comment side panel is closed.

```suggestion
                  container.matches('[data-form-side="comments"].form-side--comments [data-focus-target="true"]')
```